### DISCUSSION TITLE
Help adding a button to a URL and shortcut to Music Assistant

### DISCUSSION THREAD

**Discussion by lestat1977uk** [kalkih/mini-media-player#875] [Q&A]
Hi All, 

First of all, this is a great card, nice work.

I'm quite new to adding to cards via Yaml, I'm just trying to add a link back to Music Assisnt, so essentially I want to keep the normal click behaviour of the card, but add an extra button that will link to Music Assisnt.

I have this but it takes over the default behaviour.

type: custom:mini-media-player
volume_stateless: true
group: false
entity: media_player.studycastairreceiver_dlna_2
artwork: full-cover
info: scroll
source: full
sound_mode: full
replace_mute: stop
shortcuts:
  columns: 1
  type: button
tap_action:
  action: navigate
  navigation_path: /d5369777_music_assistant/ingress
icon: mdi:music-box-multiple-outline

Thanks in advance for any help.

**Comment by cpmarexel2f** [comment]
**1. Developer tools -> Actions and**
action: **music_assistant.get_library**
data:
  config_entry_id: 01JP5CB82H7K7WV100AS88522M
  media_type: playlist

**This is where you will see your Music Assistant playlists.**

For example:
items:
  - media_type: playlist
    uri: **library://playlist/16**
    name: 80s Pop Rock
    version: ""
    image: https://i.scdn.co/image/ab67706f0000000262706e75f2ffbd8d734668b8
    favorite: true
    explicit: null
  - media_type: playlist
    uri: **library://playlist/30**
    name: An Elegant Affair
    version: ""
    image: **https://i.scdn.co/image/ab67706f00000002616bc1a88070b930ef359a9e**
    favorite: true
    explicit: null

**2. Call up your list within the player**

  buttons:
    - image: >-
        **https://i.scdn.co/image/ab67706f00000002616bc1a88070b930ef359a9e**
      type: service
      id: media_player.play_media
      data:
        entity_id: media_player.arylic_up2stream_ma
        media_content_id: **library://playlist/30**
        media_content_type: playlist


I hope I helped you!!

**Reply by jimmyfitz** [reply]
This is helpful, to create a dedicated shortcut to launch something you already know, but I can't work out how to use this to pop-up a media-browser (i.e. turn the results of the get_library into something that you can then click)... any thoughts?  I've seen other implementations that look great, but I use mini-media-player extensively for its minimalist look and I'm trying not to break that.
